src/Sulu/Bundle/ContactBundle/Command/AccountRecoverCommand.php

```diff
@@ -17,15 +17,15 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 
 /**
  * Command for recovering nested tree of accounts.
  * This command is fixing wrong left/right and depths (see -d) assignments of the nested tree.
  */
+#[AsCommand(name: 'sulu:contacts:accounts:recover')]
 class AccountRecoverCommand extends Command
 {
-    protected static $defaultName = 'sulu:contacts:accounts:recover';
-
```

Comment:
Would document this in the `UPGRADE.md` that `$defaultName` of the commands where removed. As it would be real edge case single line is enough, just if somebody really did access it which I think is not the case.